### PR TITLE
Patch up google carousel metadata

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @curioussavage and also @xilvar

### CODE REVIEW

File: src/views/pages/listing.jsx [hunk 1]
```diff
@@ -147,8 +147,12 @@ class ListingPage extends BasePage {
   render() {
     let { data, editing, loadingMoreComments, linkEditError } = this.state;
     let { app, apiOptions, commentId, ctx, token, sort, subredditName,
-         isGoogleCrawler, origin } = this.props;
+         isGoogleCrawler } = this.props;
 
+    let { origin } = this.props.config;
+    let { url } = ctx;
+
+    console.log(this.props.ctx, this.props.config);
```
Comment (curioussavage) [inline src/views/pages/listing.jsx]: console.log here

**Comment by curioussavage** [comment]
:nail_care:

**Comment by curioussavage** [comment]
is it possible to fake a scraper to test?

**Comment by ajacksified** [comment]
@curioussavage yup - in chrome dev tools, if you open up the 'mobile phone view', you can create a custom UA. Try `Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html`

**Comment by ajacksified** [comment]
:haircut:

**Comment by curioussavage** [comment]
:fish:

**Pull request merged**